Source/Core/Common/x64Emitter.h

```diff
@@ -430,6 +430,14 @@ class XEmitter
   void PUSHF();
   void POPF();
 
+  // Intel CPU microcode migitations that fix Intel erratum SKX102 affect the caching of certain
+  // jmp/call/ret instructions: If such instructions (or such macrofused instruction pairs) span a
+  // 32-byte boundary, they will not be cached in the host CPU's icache. In order to reduce the
```

Comment:
I believe they are still cached in L1i cache, but not in the µOP cache (sometime called the L0i cache)